core/commanddefinition.js

```diff
@@ -162,12 +162,8 @@
  */
 
 /**
- * Provides command's label string. Label is define with few fallbacks in specific order described below:
- *
- * 1. If {@link CKEDITOR.command#getLabel} method exists, it is used to provide label,
- * 2. If {@link CKEDITOR.commandDefinition#label} atribute exists, it is used to provide label.
- * 3. If {@link CKEDITOR.command#uiItems} is non empty, then label of first array's item is used as label
- * 4. Empty string.
+ * Providing this method will overwrite {@link CKEDITOR.command#getLabel}, what allows on better customization
```

Comment:
My previous comment might be misleading a little, sorry for that. You moved the description to the right place as I suggested, but still:

> Providing this method will overwrite {@link CKEDITOR.command#getLabel}

Providing this method will not overwrite `CKEDITOR.command#getLabel`. The provided method will be just used by `CKEDITOR.command#getLabel` to return a label. I suggest to update the description to something like:

> If present, this method will be used to provide command's label. It allows customizing label for a specific command.

Same for `getKeyDescription` description.